src/modules/launcher/Plugins/Microsoft.PowerToys.Run.Plugin.PowerToys/Components/Utility.cs

```diff
@@ -0,0 +1,90 @@
+﻿// Copyright (c) Microsoft Corporation
+// The Microsoft Corporation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Generic;
+using System.Threading;
+using System.Windows.Input;
+using Common.UI;
+using interop;
+using Wox.Infrastructure;
+using Wox.Plugin;
+
+namespace Microsoft.PowerToys.Run.Plugin.PowerToys.Components
+{
+    public class Utility
+    {
+        public UtilityKey Key { get; }
+
+        public string Name { get; }
+
+        public bool Enabled { get; set; }
+
+        public Utility(UtilityKey key, string name, bool enabled)
+        {
+            Key = key;
+            Name = name;
+            Enabled = enabled;
+        }
+
+        public Result CreateResult(MatchResult matchResult)
+        {
+            return new Result
+            {
+                Title = Name,
+                SubTitle = "PowerToys Utility",
```

Comment:
Not sure if we need to translate "Utility" and if we need a second category "Settings Page"? 

Having the settings page as own results allows faster opening, more flexibility and showing it even if the utility is disabled.